### PR TITLE
Update ownership and emeritus state

### PR DESCRIPTION
**Pull request by seokho-son**
### Describe your changes
This PR updates overall ownership and emeritus state. (collected all recent requests)

#### Maintainers
Based on #2268 and Slack & Monthly meeting consensus. 
- Move @CathPag and @JasonMorgan to emeritus maintainer.
- Remove @CathPag and @JasonMorgan from the codeowner so that they are not notified for review. (but, they have still authority to merge PRs)

> Seokho Son
> A simple thank you doesn't seem fitting for [@Catherine](https://cloud-native.slack.com/team/U5U4E68P4). In fact, she's the main maintainer who led and sphread the Glossary project, and friend who "dragged" me in here. On behalf of all Glossary contributors and Cloud Native users, I want to say a huge, huge thank you! 
> Also, I would like to keep Catherine as an emeritus maintainer, someone we can occasionally turn to for advice as she passes by. (also to welcome whenever she is available to come back)

#### Localization approvers
- Add approvers for Russian Team based on #2220 
  - [x] @shurup 
  - [x] @kirkonru 
- Add approvers for Traditional Chinese Team based on #2321
  - [x] @hwchiu 
  - [x] @johnlinp 
- Add approver for Japanese Team based on https://github.com/cncf/glossary/issues/1422#issuecomment-1654752535
  - [x] @yuichi-nakamura 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
After we reach a consensus on this PR, I am going to open a PR to cncf/people. (https://github.com/cncf/people/pulls?q=is%3Apr+is%3Aclosed+glossary)

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Comment by seokho-son** [comment]
To candidates nominated for localization approver by localization teams,

Hello, @shurup, @kirkonru, @hwchiu, @johnlinp, @yuichi-nakamura

We invite you to review the [Policy for Localization Approvers (To Be Confirmed by Candidates)](https://github.com/cncf/glossary/discussions/723).

After reviewing, please confirm your understanding of the policy by leaving a comment on this Pull Request.

Thank you for joining this important role!! Welcome!

**Review by nate-double-u — APPROVED** [review]
Thanks so much to everyone who is stepping away, and welcome everyone who is joining or stepping into a new role 👋🏻

**Review by ydFu — APPROVED** [review]
LGTM!Welcome the new member's addition.
I have confirmed that in this pull request, the GitHub ID of the newly added member matches the issue that was requested.

**Comment by kirkonru** [comment]
Hello and thank you for your kind words! I hereby confirm that I have read and understood the information contained in the Policy for Localization Approvers.

**Comment by shurup** [comment]
Thanks for creating this PR! I've read the Policy and would like to confirm my understanding of it.

**Comment by hwchiu** [comment]
Thanks for your effort I have reviewed the Policy and understood the information and notices.

**Comment by yuichi-nakamura** [comment]
Thank you for the PR ! I have reviewed and will comply with the policy.

**Comment by johnlinp** [comment]
Thank you for the PR. I have reviewed and understood the policy!

**Comment by seokho-son** [comment]
Thank you all! 

I've created PR to sync up. https://github.com/cncf/people/pull/286 :)

**Comment by seokho-son** [comment]
https://github.com/cncf/people/pull/286 has been merged.!

Waiting for additional approval from maintainers. :)

**Comment by iamNoah1** [comment]
thanks everyone

**Pull request merged**
